src/convertToObject.js

```diff
@@ -12,7 +12,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const newObj = {};
+
+  sourceString.split(';').map(element => {
```

Comment:
```suggestion
  sourceString.split(';').forEach(element => {
```
так як ти не використовуєш результат map а просто проходишся циклом то треба використовувати forEach в такому разі